.github/workflows/pr-fix-agent.yml

```
@@ -0,0 +1,269 @@
+          def analyze_with_model(model, error_file, error_type):
+              """Analyze errors using local Ollama model"""
+              try:
+                  with open(error_file, 'r') as f:
+                      errors = f.read()[:4000]  # Limit context
+
+                  if not errors or len(errors) < 10:
+                      return None
+
+                  prompt = f"""You are a Python expert. Fix these {error_type} errors:
+
+{errors}
```

actionlint (1.7.10)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)